source/CMakeLists.txt

```diff
@@ -219,6 +219,7 @@ set(SPIRV_SOURCES
   ${CMAKE_CURRENT_SOURCE_DIR}/util/bitutils.h
   ${CMAKE_CURRENT_SOURCE_DIR}/util/bit_stream.h
   ${CMAKE_CURRENT_SOURCE_DIR}/util/bit_vector.h
+  ${CMAKE_CURRENT_SOURCE_DIR}/util/cli_consumer.h
```

Comment:
Do we want this added to the general library? it's going to get shipped to folks that are just using libspirv when it's only used for the tools?

Reply:
Okay moved. Let me know if I should have changed the namespaces for the consumer as well.  I could not think of a good thing to change it to.

Reply:
Awesome, thanks. I think spvtools::utils makes sense as the namespace.